app/retail/templates/home/index.html

```diff
@@ -275,7 +264,7 @@ <h2>{% trans "Get to know us better" %}</h2>
         </div>
       </div>
     </div>
-    <div id="articles">
+    <div id="articles" class="articles">
```

Comment:
@MaxStalker hmm could we keep it as an `id` ? 
Unless you see a reason where this would be reused again elsewhere

Reply:
In this particular case ids were used for styling - which is "no-no", since it creates more problems in the future. I've left ids there in case there are links elsewhere that reference them.
We can refactor "articles" to "home_articles" (or something similar)  if you think this might introduce name collisions later.